drag_and_drop_v2/public/js/drag_and_drop.js

```diff
@@ -301,12 +301,15 @@ function DragAndDropTemplates(configuration) {
                         h('h2.modal-window-title#'+labelledby_id, gettext('Keyboard Help'))
                     ]),
                     h('div.modal-content', [
-                        h('p', gettext('You can complete this problem using only your keyboard.')),
+                        h('p', {className:'sr'}, gettext('This is a screen reader-friendly problem')),
+                        h('p', gettext('You can complete this problem using only your keyboard by following the guidance below:')),
```

Comment:
Should we move this paragraph down one line so that it's the last paragraphs before the `ul` element?
It's a bit weird to end the paragraph with "... following the guidance below:" and then there's another paragraph between this sentence and the list of guidance items.